resources/dicts/thoughts/alive/kitten.json

```diff
@@ -52,7 +52,7 @@
         "id": "kit_med_parent",
         "thoughts": [
             "Is visiting the medicine den to see {PRONOUN/m_c/poss} parent, r_c",
-            "Is being teased for having codebreakers as parents",
+            "Is being teased for having boring medicine cats as parents",
             "Wonders if {PRONOUN/m_c/subject}'ll receive prophecies like r_c",
             "Wonders if StarClan hates {PRONOUN/m_c/object} for being a medicine cat's kit",
```

Comment:
this one also needs to be removed!

Reply:
dont know how i completely missed that! fixing now!

Reply:
Thought you meant there was an extra iteration of the codebreaker one. Removed and will consider a way to re-add to bullying kit in the future